app/controllers/HomeController.scala

```diff
@@ -84,6 +95,20 @@ class HomeController @Inject()(db: Database, cc: ControllerComponents, service:
     service.close(queueId)
   }
 
+  def queues = Action.async {
+    val patient = new Patient(2, "Foo", "Bar")
+    val result = for {
```

Comment:
somethink like queues.map(q =>Ok(views.html.queues(q)) should have the same result and it's more readable

Reply:
You're right